pages/AI-resources.md

```diff
@@ -62,3 +61,13 @@
 |  2. Git and GitHub crash course (freecodecamp) | <center> [![Git and GitHub crash course (freecodecamp)](https://i.ytimg.com/vi/RGOj5yH7evk/mqdefault.jpg "Git and GitHub crash course (freecodecamp)")](https://www.youtube.com/watch?v=RGOj5yH7evk "Git and GitHub crash course (freecodecamp)") <center>  | [Here you go](https://www.youtube.com/watch?v=RGOj5yH7evk)
 |  3. Git for Professionals (freecodecamp) | <center> [![Git for Professionals (freecodecamp)](https://i.ytimg.com/vi/Uszj_k0DGsg/mqdefault.jpg "Git for Professionals (freecodecamp)")](https://www.youtube.com/watch?v=Uszj_k0DGsg "Git for Professionals (freecodecamp)") <center>  | [Here you go](https://www.youtube.com/watch?v=Uszj_k0DGsg)
 ---
+
+## Problem Solving Challenges
+
+|Description | Preview   | Link to get started with it   |
+| :------------: | :------------: | :------------: |
+|  1. Google Hash Code 2020 | <center> [![Complete Git and GitHub Tutorial](https://i.ytimg.com/vi/Mewe008IGl0/mqdefault.jpg "Google Hash Code 2020")](https://www.youtube.com/watch?v=Mewe008IGl0 "Google Hash Code 2020") <center>  | [Here you go](https://www.youtube.com/watch?v=Mewe008IGl0)
```

Comment:
@elidakirigo, You've put the resources which are to be inside a playlist, please remove these single resources and if there is a whole playlist of hashCode on google, Please update it with the whole playlist here

Reply:
okay @hasnainmakada-99